From 41b62937fbff01c0bb62f94e471bac9d865642e0 Mon Sep 17 00:00:00 2001
From: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Date: Mon, 26 Aug 2019 15:00:50 -0700
Subject: [PATCH 0660/1852] v4l: xilinx: vip: Use the right media code for 422
 16bit formats

commit 45ee41a955a1940eaf9b1f5fb4efa7c6aad77ec6 from
https://github.com/Xilinx/linux-xlnx.git

This patch corrects the media code used for YUV 422 16bit formats.

Signed-off-by: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vip.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vip.c b/drivers/media/platform/xilinx/xilinx-vip.c
index 5de2f90df593..a9559a518938 100644
--- a/drivers/media/platform/xilinx/xilinx-vip.c
+++ b/drivers/media/platform/xilinx/xilinx-vip.c
@@ -56,9 +56,9 @@ static const struct xvip_video_format xvip_video_formats[] = {
 	  1, 16, V4L2_PIX_FMT_X212, 2, 1, 2, 1 },
 	{ XVIP_VF_YUV_422, 12, NULL, MEDIA_BUS_FMT_UYVY12_1X24,
 	  1, 16, V4L2_PIX_FMT_X212M, 2, 2, 1, 1 },
-	{ XVIP_VF_YUV_422, 16, NULL, MEDIA_BUS_FMT_UYVY12_1X24,
+	{ XVIP_VF_YUV_422, 16, NULL, MEDIA_BUS_FMT_UYVY16_2X32,
 	  2, 16, V4L2_PIX_FMT_X216, 2, 1, 2, 1 },
-	{ XVIP_VF_YUV_422, 16, NULL, MEDIA_BUS_FMT_UYVY12_1X24,
+	{ XVIP_VF_YUV_422, 16, NULL, MEDIA_BUS_FMT_UYVY16_2X32,
 	  2, 16, V4L2_PIX_FMT_X216M, 2, 2, 1, 1 },
 	{ XVIP_VF_YUV_444, 8, NULL, MEDIA_BUS_FMT_VUY8_1X24,
 	  3, 24, V4L2_PIX_FMT_VUY24, 1, 1, 1, 1 },
-- 
2.31.1

